packages/vm/src/wasm_backend/mod.rs

```diff
@@ -5,3 +5,5 @@ mod limiting_tunables;
 
 pub use compile::compile;
 pub use engine::{make_compiling_engine, make_runtime_engine};
+#[allow(unused_imports)]
+pub use limiting_tunables::LimitingTunables;
```

Comment:
This was fixed on main, no diff needed here (https://github.com/CosmWasm/cosmwasm/pull/1994/files)